sdk/servicebus/azure-messaging-servicebus/src/test/resources/xml/CreateQueueEntry.xml

```diff
@@ -0,0 +1,27 @@
+<entry xmlns="http://www.w3.org/2005/Atom">
+  <content type="application/xml">
+    <QueueDescription xmlns="http://schemas.microsoft.com/netservices/2010/10/servicebus/connect">
+      <LockDuration>PT10M</LockDuration>
+      <MaxSizeInMegabytes>1028</MaxSizeInMegabytes>
+      <RequiresDuplicateDetection>false</RequiresDuplicateDetection>
+      <RequiresSession>true</RequiresSession>
+      <DeadLetteringOnMessageExpiration>false</DeadLetteringOnMessageExpiration>
+      <MaxDeliveryCount>5</MaxDeliveryCount>
+      <EnableBatchedOperations>true</EnableBatchedOperations>
+      <AuthorizationRules>
+        <AuthorizationRule xmlns:ns0="http://www.w3.org/2001/XMLSchema-instance" ns0:type="SharedAccessAuthorizationRule">
+          <ClaimType>SharedAccessKey</ClaimType>
+          <ClaimValue>None</ClaimValue>
+          <Rights>
+            <AccessRights>Send</AccessRights>
+          </Rights>
+          <KeyName>test-name</KeyName>
+          <PrimaryKey>/4jWkzKBFwO0VemXmUYtBnXJ3Me+saF8hQPE5HzJ/hg=</PrimaryKey>
+          <SecondaryKey>NsNSEwj//k4ShF7wHsRHj54HsjnyWvyjmfCd3tfXR2Y=</SecondaryKey>
```

Comment:
Are we supposed to add these keys here ?

Reply:
Yeah. They're keys to deleted sb instances now. I can change them.